### PR TITLE
Improve standard visibility of StopArrivalView's minutes

### LINKED ISSUE #506: Departure times are not adjusting with Dynamic Type
> The estimated time of arrival on the new OneBusA app is green on white and much smaller text size so makes it harder for users with astigmatism and farsightedness like me to read.
> 
> I’ve included another app screenshot which uses black on white and slightly bolder text so easy to read.

image1

image2

### LINKED ISSUE #508: Stop departure times are harder to read than they used to be
User feedback:

> My phone seem to have just updated to a newer version about two weeks ago. Unfortunately the numbers showing the minutes until the next bus are now much smaller and a thinner font that is very hard to read in the daylight. If there is any way to change it back to a bold and/or larger font it would be super helpful.  Thanks for making a great app

### PR DESCRIPTION
**Pull request by ualch9**
"Standard" means regardless of iOS accessibility settings.
Fixes #508 and #506

| Before (no `.headline` font) | After (yes `.headline` font) |
| -- | -- |
|  |  |

### CODE REVIEW

File: OBAKitCore/Theme/Theme.swift [hunk 1]
```diff
@@ -130,8 +130,10 @@ public class ThemeColors: NSObject {
         departureEarly = .systemRed
         departureEarlyBackground = .systemRed
 
-        departureOnTime = .systemGreen
-        departureOnTimeBackground = .systemGreen
+        // Hex #129900 -- UIColor.systemGreen is too bright for small text, which StopViewController uses.
+        // See #506 and #508 for user feedback.
+        departureOnTime = UIColor(red: 0.07, green: 0.60, blue: 0.00, alpha: 1.00)
```
Comment (aaronbrethorst) [inline OBAKitCore/Theme/Theme.swift:135]: How’s this look in dark mode?

**Review by aaronbrethorst — COMMENTED** [review]
One Q for you before merging.

File: OBAKitCore/Theme/Theme.swift (hunk 1 above)
Comment (ualch9) [reply OBAKitCore/Theme/Theme.swift:135]: not good 👎

**Comment by ualch9** [comment]
| Light mode | Dark mode |
| -- | -- |
| light mode | dark mode |

**Review by aaronbrethorst — APPROVED** [review]
🎉

**Pull request merged**